scope.go

```diff
@@ -429,7 +437,8 @@ func (s *scope) Snapshot() Snapshot {
 		ss.cm.RLock()
 		for key, c := range ss.counters {
 			name := ss.fullyQualifiedName(key)
-			snap.counters[name] = &counterSnapshot{
+			id := ss.uniqueID(key, tags)
```

Comment:
Perhaps just pass `name` in here so you don't need to call `fullQualifiedName` another time in `uniqueID`?